Sources/Sentry/Profiling/SentryLaunchProfiling.m

```diff
@@ -153,9 +155,9 @@
     SENTRY_LOG_INFO(@"Starting app launch trace profile at %llu.", getAbsoluteTime());
     sentry_isTracingAppLaunch = YES;
     sentry_launchTracer =
-        [[SentryTracer alloc] initWithTransactionContext:sentry_context(tracesRate)
+        [[SentryTracer alloc] initWithTransactionContext:sentry_context(tracesRate, @1.0)
```

Comment:
Is setting the random value to `1.0` the correct way? Should we set it to something < 1.0?

Reply:
We store the `SentryLaunchProfileConfig` to disk

https://github.com/getsentry/sentry-cocoa/blob/70c49ca151ccd1a5763b1e1d316448cd5ebad948/Sources/Sentry/Profiling/SentryLaunchProfiling.m#L48-L53

We need to read the sampleRand from the stored `SentryLaunchProfileConfig`

https://github.com/getsentry/sentry-cocoa/blob/70c49ca151ccd1a5763b1e1d316448cd5ebad948/Sources/Sentry/Profiling/SentryLaunchProfiling.m#L133-L137

The FileManager should already store that info to the disk and load it to the dict here
https://github.com/getsentry/sentry-cocoa/blob/70c49ca151ccd1a5763b1e1d316448cd5ebad948/Sources/Sentry/SentryFileManager.m#L803-L816

Reply:
Sorry, I missed that you need to add new key similar to kSentryLaunchProfileConfigKeyTracesSampleRate and update the method here

https://github.com/getsentry/sentry-cocoa/blob/70c49ca151ccd1a5763b1e1d316448cd5ebad948/Sources/Sentry/Profiling/SentryLaunchProfiling.m#L165-L187

Reply:
Added it.